composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.3.2|^7.0",
-        "symfony/framework-bundle": "~2.0|^3.0",
+        "symfony/framework-bundle": "~2.0|^3.1",
```

Comment:
You need to use `symfony/event-dispatcher:^2.8`:

```bash
composer require symfony/event-dispatcher:^2.8
```

This is because this bundle uses [omnipay](https://github.com/thephpleague/omnipay), which in turn uses Guzzle3, which in turn requires `symfony/event-dispatcher:~2.1`. From omnipay's docs:

> If you are using Symfony 3 (or Symfony 3 components), please note that Omnipay 2.x still relies on Guzzle3, which in turn depends on symfony/event-dispatcher 2.x. This conflicts with Symfony 3 (standard install), so cannot be installed. Development for Omnipay 3.x is still in progress at the moment.

> If you are just using the Symfony 3 components (eg. stand-alone or Silex/Laravel etc), you could try to force the install of symfony/event-dispatcher:^2.8, which is compatible with both Symfony 3 components and Guzzle 3.




BTW, not my bundle :)